pkg/testing/tools/artifacts_api.go

```diff
@@ -119,6 +122,10 @@ func WithUrl(url string) ArtifactAPIClientOpt {
 	return func(aac *ArtifactAPIClient) { aac.url = url }
 }
 
+func WithCDNUrl(url string) ArtifactAPIClientOpt {
```

Comment:
[Suggestion]
I won't put it as a blocker, but so far I haven't seen any reference to the artifacts API as _CDN_. From my perspective using _CDN_ is at least counter intuitive, if not confusing.

I'd suggest to use `WithURL` as there is just one URL to be configured in the client

Reply:
I'm not sure I understand your comment. There is the artifact API URL and there is the CDN URL where we publish our artifacts for the customers once versions are released. We need both at the same time and we already have `WithURL` option for the artifact API.

Reply:
oh, I mixed things up and I forgot there was already another `WithURL`. Never mind